platform/nodejs/package.json

```diff
@@ -23,8 +23,8 @@
     "url": "https://github.com/gorhill/uBlock/issues"
   },
   "homepage": "https://github.com/gorhill/uBlock#readme",
-  "dependencies": {
-    "rollup": "^2.55.1",
-    "rollup-plugin-terser": "^7.0.2"
+  "engines": {
+    "node": ">=14.0.0",
```

Comment:
I have tested it on these versions, which are already a little old.

This field is [advisory](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines) and gives warnings only.